package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@sylius-ui/admin": "file:src/Sylius/Bundle/AdminBundle",
     "chart.js": "^3.7.1",
+    "infinite-tree": "^1.17.1",
```

Comment:
Should this dependency be added only to `src/Sylius/Bundle/AdminBundle/package.json`?